docs/managing-wallets.md

```diff
@@ -21,6 +21,7 @@ in the future
 - No indication that changes to the code are not properly tested
 - Wallet was publicly announced and released since at least 3 months
 - No concerning bug is found when testing the wallet
+- Provides a bug reporting method on the website and in the app
```

Comment:
We should determine what qualifies as a bug reporting method in order for the requirement to be satisfied and to provide clarity to the wallet provider. For example, whether it's a specific email address that's dedicated to bugs, a link to a bug bounty / responsible disclosure program, or something else.

Reply:
We have been accepting anything that allow contacting the developer and I'm OK with that.  What do you think?

Reply:
Since we've revised the requirement to "and/or" (app/website) for the method, SGTM; thinking we can just review on a case by case basis. Was previously concerned that if it needed to be in both places it might not be clear exactly what we are looking for in order to satisfy the requirement (it sounded a bit more stringent if it had to be in all places).